CHANGELOG.md

```diff
@@ -1,5 +1,39 @@
 # Optimizely Java X SDK Changelog
 
+## 3.0.0
+February 11, 2019
+
+The 3.0 release improves event tracking and supports additional audience targeting functionality.
+### New Features:
+* Event tracking:
+    * The Track method now dispatches its conversion event _unconditionally_, without first determining whether the user is targeted by a known experiment that uses the event. This may increase outbound network traffic.
+    * In Optimizely results, conversion events sent by 3.0 SDKs are automatically attributed to variations that the user has previously seen, as long as our backend has actually received the impression events for those variations.
+    * Altogether, this allows you to track conversion events and attribute them to variations even when you don't know all of a user's attribute values, and even if the user's attribute values or the experiment's configuration have changed such that the user is no longer affected by the experiment. As a result, **you may observe an increase in the conversion rate for previously-instrumented events.** If that is undesirable, you can reset the results of previously-running experiments after upgrading to the 3.0 SDK.
+    * This will also allow you to attribute events to variations from other Optimizely projects in your account, even though those experiments don't appear in the same datafile.
```

Comment:
We do not support this in the product yet. @nchilada I am probably late to the conversation, but I am assuming this is fine to put in?

Reply:
Yeah, I think this is okay. I believe this is already exposed in data export, and we're planning to support this in the results page in Q1.

And I used future tense just in case, despite being unhappy about the jarring tense 😂
> This will ...